index.html

```diff
@@ -1724,6 +1724,78 @@ <h4>CredentialRegistry</h4>
         </pre>
       </section>
 
+      <section>
+        <h4>OpenID4 Verifiable Credential Issuance</h4>
+        <p>The <code>OID4VCI</code> service allows publication of the <a href="https://openid.net/openid4vc/">OpenID for Verifiable Credential Issuance (OID4VCI)</a> domain, where the <code>.well-known</code> files (<code>oauth-authorization-server</code> & <code>openid-credential-issuer</code>) can be found. These files MUST contain the endpoints and meta data for <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenID for Verifiable Credential Issuance (OpenID4VCI)</a>.</p>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenID4VCI</a>
+              </td>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/context/openid4vc">OpenID4VCI</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of service and serviceEndpoint properties">
+{
+  ...
+  "service": [
+    {
+      "id": "did:example:123#oid4vci",
+      "type": "OID4VCI",
+      "serviceEndpoint": "https://did.example.com/"
+    }
+  ]
+}
+        </pre>
+      </section>
+
+      <section>
+        <h4>OpenID4 Verifiable Presentation</h4>
+        <p>The <code>OID4VP</code> service allows publication of the <a href="https://openid.net/openid4vc/">OpenID for Verifiable Presentation (OID4VP)</a> authorization endpoint, which is the entry point for the <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html">presentation flow</a>.</p>
+        <table class="simple" style="width: 100%;">
```

Comment:
I would support an URI or Object for the service endpoint.

IF URI then `id` is the OAuth2 authorization server issuer identifier (URI) as per https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-wallet-metadata-authorizati and RFC8414.

IF Object, then the service endpoint contains the Wallet Metadata according to https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-wallet-metadata-authorizati, and an OPTIONAL `id` as described above.

Reply:
Only `id` as per my comment above.

Reply:
Integrated in https://github.com/w3c/did-spec-registries/pull/513/commits/e7c2827d78b79bbb4af97036673d4df18b116518

Can you confirm the correctness of the changes?

Reply:
Merged your suggestions :)